src/reducers/account/index.js

```diff
@@ -61,6 +63,15 @@ const requestResultReducer = (state, { error, ready, payload, meta }) => {
     if (!meta || !meta.successCode) {
         return state
     }
+
+    // any generic store update to state (e.g. the flag requestPending)
```

Comment:
not sure how this is supposed to be used

Reply:
@mattlockyer ?

Reply:
Missed comment sorry. I did not understand redux logic for updating the store. I spent hours trying to figure out which reducers were called when I called store.dispatch(someAction). Finally, I settled on this pattern to update the account store. Is there a better way?

Reply:
All of the reducers specified in the list get called, not sure I understand what is the question.
Reducers defined using `handleActions({` basically have actions as keys and corresponding reducer functions as values.

I'd prefer we don't have "generic" updates to store as then it more or less defeats the purpose of using Redux.

Reply:
ok all fixed